packages/block-library/src/navigation-submenu/index.php

```diff
@@ -163,7 +163,7 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 	}
 
 	$show_submenu_indicators = isset( $block->context['showSubmenuIcon'] ) && $block->context['showSubmenuIcon'];
-	$open_on_click = isset( $block->context['openSubmenusOnClick'] ) && $block->context['openSubmenusOnClick'];
+	$open_on_click           = isset( $block->context['openSubmenusOnClick'] ) && $block->context['openSubmenusOnClick'];
 	$open_on_hover_and_click = isset( $block->context['openSubmenusOnClick'] ) && ! $block->context['openSubmenusOnClick'] &&
```

Comment:
I wonder if it should just be called $open_on_hover? Setting focus on the separate dropdown chevron and pressing Enter would also open it. But for mouse users, it would open only on hover, right?

Reply:
Sure, but this variable is only true when menus open on hover and click, not on hover and focus. We're setting a classname specifically for the hover and click combo so we can _not_ open the submenu when the parent link is focused. (See CSS rule [here](https://github.com/WordPress/gutenberg/pull/33775/files#diff-d66a4ed73d0d50a799547b78db235a4bad3428a0decb8895752032cd849d01f0R183).)